packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -197,7 +197,12 @@ export const TrackListItem = ({
           />
         ) : isActive && !isDeleted ? (
           <View style={styles.playButtonContainer}>
-            <TablePlayButton playing paused={!isPlaying} hideDefault={false} />
+            <TablePlayButton
+              playing
+              paused={!isPlaying}
+              hideDefault={false}
+              onPress={onClickTrack}
```

Comment:
prob not have "click" in the function name, so maybe onPressTrack or handlePressTrack?